tests/Fixer/PhpUnit/PhpUnitExpectationFixerTest.php

```diff
@@ -305,7 +305,7 @@ function testFnc%d()
             zzz();
         }
 '
-;
+        ;
```

Comment:
why this change? is it expected?

Reply:
Yep, the new `StatementIndentationFixer` reindents all lines unless they are part of a multiline statement. It considers `;` to *not* be part of the expression, e.g.:

```diff
-  $foo =
-            true
-             || false
-            ;
+$foo =
+          true
+           || false
+;
```
Reindenting the lines that are part of a statement should be another rule IMO, out of the scope of this PR.

Reply:
not sure if it's out of scope - like one would say it's a change of behaviour of existing fixer.

is this new behaviour (the extra spaces) are improving the rule or not?

Reply:
looking good